analysis/parser/RawTextParser/src/test/resources/cc_projects/project_4.cc.json

```diff
@@ -1,5 +1,5 @@
 {
-  "projectName": "foo",
```

Comment:
How come the test (data) for the raw test parser is changed?

Reply:
Because the unit tests were failing. We just never noticed, because we never executed the unit tests.